css/index.html

```diff
@@ -0,0 +1,108 @@
+        </div>
+        <p>En PlanIT tambien puedes encontrar regalos para tu familia y amigos.</p>
+      </div>
+    </div>
+    <div class="banner-images">
+      <div class="events-gifts">
+        <img src="./assets/images/image1.jpeg" alt="image1" class="banner-image1">
+        <img src="./assets/images/image2.jpeg" alt="image2" class="banner-image2">
+      </div>
+      <div class="catering-merchan">
+        <img src="./assets/images/image3.jpeg" alt="image3" class="banner-image3">
+        <img src="./assets/images/image4.jpeg" alt="image4" class="banner-image4">
+      </div>
+    </div>
+  </section>
+
+  <div class="partner-companies">
+    <h5>Trabajamos dia a dia junto a empresas que confian en nosotros</h5>
+    <div class="partner-companies-logos">
+      <img src="./assets/images/logo1.jpeg" alt="logo1" class="partner-companies-images">
+      <img src="./assets/images/logo2.jpeg" alt="logo2" class="partner-companies-images">
+      <img src="./assets/images/logo3.jpeg" alt="logo3" class="partner-companies-images">
+      <img src="./assets/images/logo4.jpeg" alt="logo4" class="partner-companies-images">
+      <img src="./assets/images/logo5.jpeg" alt="logo5" class="partner-companies-images">
+      <img src="./assets/images/logo6.jpeg" alt="logo6" class="partner-companies-images">
+      <img src="./assets/images/logo7.jpeg" alt="logo7" class="partner-companies-images">
+      <img src="./assets/images/logo8.jpeg" alt="logo8" class="partner-companies-images">
+
+      <img src="./assets/images/logo9.jpeg" alt="logo9" class="partner-companies-images">
+      <img src="./assets/images/logo10.jpeg" alt="logo10" class="partner-companies-images">
+      <img src="./assets/images/logo11.jpeg" alt="logo11" class="partner-companies-images">
+      <img src="./assets/images/logo12.jpeg" alt="logo12" class="partner-companies-images">
+      <img src="./assets/images/logo13.jpeg" alt="logo13" class="partner-companies-images">
+      <img src="./assets/images/logo14.jpeg" alt="logo14" class="partner-companies-images">
+      <img src="./assets/images/logo15.jpeg" alt="logo15" class="partner-companies-images">
+      <img src="./assets/images/logo16.jpeg" alt="logo16" class="partner-companies-images">
+    </div>
+
+    <div class="highlights">
+      <div class="highlights1">
+        <div class="highlights-AT">
+          <h3>Ahorra tiempo</h3>
+          <h4>Planifica tu ano en base a tu presupuesto y programa tus acciones. Linea directa entre proveedores y
+            empresas</h4>
+        </div>
+        <img src="./assets/images/highlight1.jpeg" alt="highlight1" class="highlights-images">
+      </div>
+      <div class="highlights1">
+        <img src="./assets/images/highlight2.jpeg" alt="highlight2" class="highlights-images">
+        <div class="highlights-AT">
+          <h3>Comprende a tu equipo</h3>
+          <h4>Obtene feedback para mejorar tus acciones en cada etapa.</h4>
+        </div>
+      </div>
+    </div>
```

Comment:
Class names here are not really clear. Also, you used highlights-AT where I can guess that AT means `ahorra tiempo` for both highlights, not only for the one saying `ahorra tiempo`. Maybe you can use something like `highlights-container`, `highlight` and `highlight-texts / highlight-description` for the names.